commands/upgrade/cluster/command.go

```diff
@@ -359,6 +361,15 @@ func upgradeCluster(args Arguments) (*upgradeClusterResult, error) {
 	return result, nil
 }
 
+func isVersionProductionReady(version string) bool {
+	semverVersion, err := semver.NewVersion(version)
+	if err != nil {
+		return false
+	}
+
+	return len(semverVersion.Prerelease()) < 1 && len(semverVersion.Metadata()) < 1
```

Comment:
What this `semverVersion.Metadata()` tell you?

Reply:
https://pkg.go.dev/github.com/Masterminds/semver/v3#Version.Metadata

It probably returns `-alpha1` for a string like `13.0.0-alpha1`.

Reply:
Or rather `alpha1` instead of `-alpha1`

Reply:
Well, here is an example: https://play.golang.org/p/Bbvs1yIxAn8

Reply:
In `1.0.0-alpha1+clang`, `alpha1` is the pre-release information, and `clang` is the custom build metadata.
So `.Prerelease()` would return `alpha1` and `.Metadata()` would return `clang`